worker/categorize.py

```diff
@@ -403,11 +404,18 @@ def _is_viable_candidate(self, phrase):
                 return False
 
             titled_words = map(lambda x: x.istitle(), words)
-            if titled_words not in [[True, False, True], [True, True,True]]:
+            if titled_words not in [[True, False, True], [True, True, True]]:
                 return False
 
         return True
 
+    def _is_nested(self, keyword, other_keywords):
+        for other_keyword in other_keywords:
+            if keyword != other_keyword and keyword in other_keyword:
```

Comment:
Should we favor longer keywords over shorter, or build in anything like that? Should "Google Maps" be chosen over "Google"? What happens now? Is it just luck of which runs first?